### DISCUSSION TITLE
Mediamtx and fail2ban in linux

### DISCUSSION THREAD

**Discussion by adzmailru** [bluenviron/mediamtx#3868] [General]
Hello, in order to write a fail2ban filter, you can start by enabling debug logging inside MediaMTX:

logLevel: debug

The log lines you're interested into are these ones:
2024/08/27 23:06:55 DEB [RTSP] [conn 127.0.0.1:44480] [s->c] RTSP/1.0 401 Unauthorized
CSeq: 2

filter file of mediamtx (/etc/fail2ban/filter.d/mediamtxxxx.local):

[Init]
maxlines = 2

[Definition]
failregex = `.* DEB \[RTSP\] \[conn <HOST>:\d+\] \[s->c\] RTSP\/1\.0 401 Unauthorized\nCSeq: 2`
ignoreregex =

**Comment by aler9** [comment]
moved to https://github.com/bluenviron/mediamtx/issues/5016
